wazuhapp/generate_wazuh_app.sh

```diff
@@ -100,6 +101,10 @@ main(){
                 READY_TO_RELEASE="yes"
             fi
             ;;
+        "-k" | "--checksum")
+            cheksum="yes"
```

Comment:
Typo => `cheksum`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/9d09627160a93091dae44ea05cd9ca3429cd8f21